examples/ZMQPP/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZMQPP)
+
+hunter_add_package(ZMQPP)
+
+find_package(ZMQPP CONFIG REQUIRED)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
Won't be better if I changed this something like this (before hunter_add_package):

```cmake
if(CMAKE_CXX_STANDARD LESS 11)
  message(FATAL_ERROR, "C++11 required for ZMQPP")
endif()
```

in this case the user will get a correct error message. Furthermore I'm not sure that add c++11 attribute to compiler flags just only for project won't cause silent or noisy link error...

Can I force this check somewhere in the ZMQPP hunter definition? (Earlier the ZMQPP's `CMakelists.txt`).